tasks/letsencrypt.yml

```diff
@@ -33,3 +33,13 @@
   command: certbot certonly -d {{ bbb_hostname }} --register-unsafely-without-email --agree-tos -n --nginx
   notify: reload nginx
   when: not certbot_certificate_file_path.stat.exists and bbb_letsencrypt_email is not defined
+
+- name: Generate test Certificates with letsencrypt expire notifications
+  command: certbot certonly -d {{ bbb_hostname }} --email {{ bbb_letsencrypt_email }} --agree-tos -n --nginx --test-cert
+  notify: reload nginx
+  when: not certbot_certificate_file_path.stat.exists and bbb_letsencrypt_email is defined and bbb_letsencrypt_staging | bool
+
+- name: Generate test Certificates without letsencrypt expire notifications
+  command: certbot certonly -d {{ bbb_hostname }} --register-unsafely-without-email --agree-tos -n --nginx --test-cert
+  notify: reload nginx
+  when: not certbot_certificate_file_path.stat.exists and bbb_letsencrypt_email is not defined and bbb_letsencrypt_staging | bool
```

Comment:
don't we want to make an  `if` for `bbb_letsencrypt_staging` which executes this when enabled and the chunk above if false? Otherwise it will also generate real certificates.